apps/preview/next/pages/showcases/Checkout/CheckoutPaymentMethod.tsx

```diff
@@ -1,50 +1,46 @@
 import { ShowcasePageLayout } from '../../showcases';
 // #region source
-import ApplePayLogo from '@assets/apple-pay-logo.svg';
-import GooglePayLogo from '@assets/google-pay-logo.svg';
-import PayPalLogo from '@assets/paypal-logo.svg';
-import VisaLogo from '@assets/visa-logo.svg';
 
 // List of payment methods
 const paymentMethods = [
   {
     label: 'Credit card',
     value: 'credit-card',
-    logo: VisaLogo.src,
+    logo: '/visa-logo.svg',
     active: true,
   },
   {
     label: 'PayPal',
     value: 'paypal',
-    logo: PayPalLogo.src,
+    logo: '/paypal-logo.svg',
     active: true,
   },
   {
     label: 'ApplePay',
     value: 'applepay',
-    logo: ApplePayLogo.src,
+    logo: '/apple-pay-logo.svg',
     active: true,
   },
   {
     label: 'GooglePay',
     value: 'googlepay',
-    logo: GooglePayLogo.src,
+    logo: '/google-pay-logo.svg',
     active: false,
   },
 ];
 
 export default function PaymentMethods() {
   return (
     <fieldset>
-      <legend className="mb-4 typography-headline-5 font-bold text-neutral-900">Payment method</legend>
-      <div className="grid grid-cols-2 gap-4 items-stretch">
+      <legend className="mb-4 font-bold typography-headline-5 text-neutral-900">Payment method</legend>
```

Comment:
question: Are you using some kind of plugin to order those classes?

Reply:
yes, it's called Headwind and at the very beginning it was decided that as a team we'll use it. I can remove it if these changes aren't necessary now :)